docs/reference/eql/eql.asciidoc

```diff
@@ -797,9 +797,12 @@ You can also manually delete saved synchronous searches using the
 
 experimental::[]
 
-The EQL search API supports <<modules-cross-cluster-search,cross-cluster
-search>>. However, the local and <<remote-clusters,remote clusters>>
-must use the same {es} version.
+The EQL search API supports <<modules-cross-cluster-search,cross-cluster search>>. However, the local and <<remote-clusters,remote clusters>> must use the same {es} version if they have versions prior to 7.17.7 (included) or prior to 8.5.1 (included).
+search>> with <<remote-clusters,remote clusters>> v 7.17.8+,
+and limited to <<ccs-supported-configurations>>.
+However, if local node has version prior to 7.17.8 or 8.5.2, <<remote-clusters,remote clusters>>
+must be the same {es} version to run <<modules-cross-cluster-search,cross-cluster
+search>>.
```

Comment:
This one is not right. It got messed up when the suggestion was applied.

Reply:
fixing